tests/unit/flow-construct/test_flow_except.py

```diff
@@ -143,25 +143,18 @@ def validate2(x, *args):
 
 @pytest.mark.parametrize('protocol', ['websocket', 'grpc', 'http'])
 def test_no_error_callback(mocker, protocol):
-    def validate2():
```

Comment:
Since we have removed the callbacks, this test doesn't make sense anymore right? Let's skip it if it fails.

Reply:
ah ok let me change this

Reply:
This test is still valid, we checdk the `on_error` is calle